ebpf/state.go

```diff
@@ -119,10 +122,13 @@ func (ns *networkState) getClients() []string {
 // Connections returns the connections for the given client
 // If the client is not registered yet, we register it and return the connections we have in the global state
 // Otherwise we return both the connections with last stats and the closed connections for this client
-func (ns *networkState) Connections(id string, latestConns []ConnectionStats) []ConnectionStats {
+func (ns *networkState) Connections(id string, latestTime uint64, latestConns []ConnectionStats) []ConnectionStats {
 	ns.Lock()
 	defer ns.Unlock()
 
+	// Update the latest known time
+	ns.latestTimeEpoch = latestTime
```

Comment:
Do we want to guard against non-monotonic updates here?  I believe I saw that if there was an error fetching the time it wouldn't be updated.

Reply:
I don't think we need to, since if there's an error fetching the time we don't return any active connections at all (just closed connections)

We don't guard against this when we expire TCP/UDP connections either, so I'd like to keep it consistent until I can dig deeper into the out-of-order + integer underflow issues we've been seeing. I have a few suspicions for whats happening (which is related to these monotonic times)